ets-demo/setup.py

```diff
@@ -0,0 +1,288 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+import os
+import runpy
+import setuptools
+import subprocess
+
+# Version information; update this by hand when making a new bugfix or feature
+# release. The actual package version is autogenerated from this information
+# together with information from the version control system, and then injected
+# into the package source.
+MAJOR = 0
+MINOR = 1
+MICRO = 0
+PRERELEASE = ""
+IS_RELEASED = False
+
+# If this file is part of a Git export (for example created with "git archive",
+# or downloaded from GitHub), ARCHIVE_COMMIT_HASH gives the full hash of the
+# commit that was exported.
+ARCHIVE_COMMIT_HASH = "$Format:%H$"
+
+# Templates for version strings.
+RELEASED_VERSION = "{major}.{minor}.{micro}{prerelease}"
+UNRELEASED_VERSION = "{major}.{minor}.{micro}{prerelease}.dev{dev}"
+
+# Paths to the autogenerated version file and the Git directory.
+HERE = os.path.abspath(os.path.dirname(__file__))
+VERSION_FILE = os.path.join(HERE, "etsdemo", "version.py")
+# etsdemo is sharing the same Git directory with traitsui
+GIT_DIRECTORY = os.path.join(HERE, "..", ".git")
+
+# Template for the autogenerated version file.
+VERSION_FILE_TEMPLATE = '''\
+# (C) Copyright 2020-2020 {company}, Inc., Austin, TX
```

Comment:
A lot of this copied from traits. I did change the copyright year start to 2020.